### ISSUE TITLE
Linked resources start closed

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/Digital-Exhibits-Issues-Features#1]

**Comment by graykr** [comment]
We'd like the option to have the linked resources default to open instead of closed. In our implementation for oral history themes and locations, we want the users of the site to easily see the linked resources -- they are intended to be the main focus of the item page in this case.

**Comment by spenserbailey** [comment]
Agreed with Krista on this one - especially for me, since it took a while to get the subjects/linked resources to work properly.
